src/kbc_modules/eaa_kbc/protocol.rs

```diff
@@ -7,6 +7,11 @@ use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 use std::vec::Vec;
 
+pub const GET_POLICY_COMMAND: &str = "Get Policy";
+pub const GET_SIGSTORE_CONFIG_COMMAND: &str = "Get Sigstore Config";
+pub const GET_GPG_KEY_RING_COMMAND: &str = "Get GPG Keyring";
```

Comment:
These appear to be identical to the `const`'s in src/kbc_modules/eaa_kbc/mod.rs, but with a "`Get `" prefix. Is there any way to generate these from the `const` values in `mod.rs` to make this relationship clearer (and ensure the code doesn't break if `mod.rs` is updated.